less/page-header.less

```diff
@@ -17,90 +12,108 @@ html {
 
   &:after {
     content: ' ';
+
     position: absolute;
+
+    visibility: hidden;
+
+    transition: left .2s ease-in-out, right .2s ease-in-out;
+
     bottom: 0;
     height: 2px;
     right: 50%;
     left: 50%;
-    background: var(--link-color);
-    visibility: hidden;
-    transition: left .2s ease-in-out, right .2s ease-in-out;
+
+    background: var(--link__color);
   }
 
   &:hover {
     &:after {
+      visibility: visible;
       right: 0;
       left: 0;
-      visibility: visible;
     }
   }
 }
 
 
-.page_header {
-  backdrop-filter: blur(12px);
+.page-header {
   position: sticky;
-  top: 0;
   z-index: 1;
-  height: var(--pageHeader-height);
-  background-color: var(--pageHeader-bg);
-  box-shadow: 0 2px 8px var(--pageHeader_box-shadow-color);
+
+  top: 0;
+  height: var(--page-header__height);
+  box-shadow: 0 2px 8px var(--page-header__box-shadow__color);
```

Comment:
It's harder to make bold statements when it comes to CSS variable names, since BEM predates CSS variables and doesn't make any naming prescriptions.

Still, I find it a bit weird to have two `__`s in a variable name for the same reasons that I do elsewhere.

Reply:
That's completely valid if you don't think we should follow BEM for CSS variable names.

We could:
* Just use singular `_` and `-` in variables, to be following BEM conventions but making them easier to type.
* We could follow some totally different convention?

Reply:
I'm just saying that two `__`s in a variable name is always wrong, just as it is in class names. But if I'm honest, I would prefer to just do hyphens here instead of making it BEM-like.

For instance, this one could be `--page-header-box-shadow-color` instead.

Reply:
I see what you are saying. I'll go ahead and fix up variable naming to just be all hyphens then. Thanks for the feedback. And I do hear your disapproval of `__` in class names as well, which I know I've included in mass on this PR. 

I personally do feel there's great value in the naming structure of BEM making things more clear, do you think it's worth considering if we want to change our delimiters between the elements of the naming convention? I know your last comment likely wasn't meant in reference to the entire changes here, but just wanting to make sure we can find something to suit both of our priorities, between structure and usability.

Reply:
Alright, all CSS variables have now been hyphenated.

Reply:
> And I do hear your disapproval of `__` in class names as well, which I know I've included in mass on this PR.

No — re-read my comments :-) I'll elaborate later if need be.

Reply:
Ahh, I see what you mean now. I totally misunderstood.
You meant having two instances of `__` as in two elements (according to BEM) within a single name. Not having two individual `_` characters.

In that case disregard the latter half of my message. But I do hope that changing the naming of variables away from BEM to include only `-` as delimiters is something you are on board with, but we can always revert the commit if need be. My bad on the misunderstanding